dChatServer/ChatServer.cpp

```diff
@@ -295,7 +297,7 @@ void HandlePacket(Packet* packet) {
 			break;
 
 		default:
-			LOG("Unknown CHAT id: %i", int(packet->data[3]));
+			LOG("Unknown/Unhandled CHAT Message id: %s (%i)", StringifiedEnum::ToString(chat_message_type).data(), chat_message_type);
```

Comment:
nit: add the explicitly unhandled ones as a fall through case statement such that if we plan to handle them, it logs Unhandled.